### ISSUE TITLE
API21上滚动条颜色异常

### ISSUE THREAD

**Issue by LiuJiangshan** [zhanghai/AndroidFastScroll#32]
直接变白色了，与背景色一样了
![捕获](https://user-images.githubusercontent.com/13770291/97944978-1a2d4480-1dc1-11eb-98c3-226ffb137978.PNG)

R.drawable.afs_thumb
```
<shape xmlns:android="http://schemas.android.com/apk/res/android"
    android:shape="rectangle"
    android:tint="?colorControlActivated">

    <size
        android:width="8dp"
        android:height="48dp" />

    <solid android:color="@android:color/white" />
</shape>
```
貌似android:tint问题，但api21是支持tint的

**Comment by zhanghai** [comment]
确认了——在 5.1（22）上是好的，但是 5.0（21）上就不见了……有些奇怪。

**Comment by zhanghai** [comment]
> DrawableCompatLollipop.java: GradientDrawable on Lollipop does not support tinting, so we'll use our compatible functionality instead
> alanv: OS bug that's been fixed for a future release. ImageView's tint attribute was updated to use Drawable.setTint() rather than Drawable.setColorFilter(), but GradientDrawable didn't support setTint(). There is not really a good workaround for this from XML, but you can call setColorFilter() from code.

**Comment by zhanghai** [comment]
Found out that [the commit adding tint support](https://android.googlesource.com/platform/frameworks/base/+/43027b7bca554818cc4223389747d661819d5706%5E%21/) (and [a subsequent fix](https://android.googlesource.com/platform/frameworks/base/+/3ad771b384b55bc0b9ee9519f457ea7093bb0b74%5E%21/)) for `GradientDrawable` actually wasn't [in Lollipop](https://android.googlesource.com/platform/frameworks/base/+log/refs/heads/lollipop-release/graphics/java/android/graphics/drawable/GradientDrawable.java), but was only available [since Lollipop MR1](https://android.googlesource.com/platform/frameworks/base/+log/refs/heads/lollipop-mr1-release/graphics/java/android/graphics/drawable/GradientDrawable.java). So although the `setTintList()` method was added to `Drawable` in API 21, `GradientDrawble` didn't override and implement it until API 22, and we have to work around it with things like `setColorFilter()`.

**Comment by zhanghai** [comment]
Fixed in 91a796a.

**Comment by zhanghai** [comment]
Released in [v1.1.5](https://github.com/zhanghai/AndroidFastScroll/releases/tag/v1.1.5).
